translations/messages.fr.xlf

```diff
@@ -1174,6 +1182,66 @@
+        <source>You do not have any pieces of this type anymore.</source>
+        <target>Vous n'avez plus de pièces de ce type.</target>
+      </trans-unit>
+      <trans-unit id="1718016291859374582" datatype="html">
+        <source>You cannot play here: this space is already full.</source>
+        <target>Vous ne pouvez pas jouer ici : cette case est déjà pleine.</target>
+      </trans-unit>
+      <trans-unit id="3011799294333722541" datatype="html">
+        <source>You cannot add more pieces to the target space, as it will contain more than 4 pieces.</source>
+        <target>Vous ne pouvez pas ajouter de pièces dans la case ciblée, car elle contiendrait plus de 4 pièces.</target>
+      </trans-unit>
+      <trans-unit id="5660375161018346302" datatype="html">
+        <source>To perform a shift, you must move the pieces to the space either directly to the left or directly to the right.</source>
+        <target>Pour déplacer des pièces du plateau, vous devez les déplacer sur la case directement à gauche ou directement à droite de la case de départ.</target>
+      </trans-unit>
+      <trans-unit id="290467566247457693" datatype="html">
+        <source>You must first select either a piece from the side, or a stack of pieces to shift on the board.</source>
+        <target>Vous devez d'abord sélectionner une pièce hors du plateau, ou une pièce étant sur une case du plateau pour la déplacer.</target>
+      </trans-unit>
+      <trans-unit id="354630056284498570" datatype="html">
+        <source>Initial board and player pieces</source>
+        <target>Plateau initial et pièces des joueurs</target>
+      </trans-unit>
+      <trans-unit id="8818359317795688141" datatype="html">
+        <source>Diam&apos;s board is a circular board made of 8 spaces. Each player has 8 pieces: 4 of one color, and 4 of another color. Initially, the board is empty. All the remaining pieces are displayed next to the board: Dark&apos;s pieces are on the left, Light&apos;s pieces are on the right.</source>
+        <target>Le plateau de Diam est un plateau circulaire composé de 8 cases. Chaque joueur possède 8 pièces : 4 d'une couleur, et 4 d'une autre couleur. Initialement, le plateau est vide. Toutes les pièces restantes sont montrées sur les côté du plateau : les pièces de Foncé sur la gauche, les pièces de Clair sur la droite.</target>
+      </trans-unit>
+      <trans-unit id="6414311869674207113" datatype="html">
+        <source>At Diam, the goal is to align two of your pieces, having exactly the same color, on diametrically opposed spaces, on top of at least another piece. Note here that Dark does not win because the dark pieces are not on top of any other piece. You&apos;re playing Light. Here, you can win by dropping one of your pieces on the leftmost space. You can do it by clicking on the corresponding piece next to the board, and then on the space you want to drop it.&lt;br/&gt;&lt;br/&gt;Do it!</source>
```

Comment:
and then on the space you want to drop it.
Le it final est ambigu, c'est bien formulé ?

Reply:
to *drop your piece* (changé)

Reply:
Je trouves toujours ça bizarre. J'ajouterais un "in which" (and then on the space **in which** you want to drop your piece)

Reply:
ok ça fait sens, ajouté